### ISSUE TITLE
`_isNumerical` implementation doesn't make sense

### ISSUE THREAD

**Issue by billinghamj** [domchristie/humps#48]
I was just reviewing this code, and noticed that the following doesn't seem to make sense:

https://github.com/domchristie/humps/blob/d612998749922a76c68d4d9c8b5ae93f02595019/humps.js#L94-L98

Perhaps this would be more in-line with the original intent?

```js
var _isNumerical = function(obj) { 
  var num = obj - 0; 
  return num === obj; 
}; 
```

That would check if the input *was* that precisely that number (including the type), but the comment states that it checks whether the value _coerces_ to a number, so perhaps it should also really be double equals, rather than triple? Or perhaps the comment should be corrected?

**Comment by seansfkelley** [comment]
> As, obviously, this code could never return false. You could replace the entire implementation with `return true` and it would behave the same way.

Not quite:

```
> NaN === NaN
false
```

Which I think is the intent, in which case it would perhaps be clearer for this function to be implemented as `return !isNaN(obj - 0);` or perhaps `return !isNaN(+obj);`.

**Comment by billinghamj** [comment]
Ah yes I guess that could be it. Definitely should be using the explicit function for that - perhaps the `Number.isNaN` one though.

**Comment by billinghamj** [comment]
Yep, there we go: https://github.com/domchristie/humps/pull/13#discussion-diff-31913422
